x/bank/simulation/msgs.go

```diff
@@ -16,11 +16,30 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/mock/simulation"
 )
 
-// SingleInputSendTx tests and runs a single msg send w/ auth, with one input and one output, where both
+// SendTx tests and runs a single msg send where both
 // accounts already exist.
-func SingleInputSendTx(mapper auth.AccountKeeper) simulation.Operation {
+func SendMsg(mapper auth.AccountKeeper, bk bank.Keeper) simulation.Operation {
+	handler := bank.NewHandler(bk)
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account, event func(string)) (action string, fOps []simulation.FutureOperation, err error) {
+		fromAcc, action, msg, abort := createSendMsg(r, ctx, accs, mapper)
+		if abort {
+			return action, nil, nil
+		}
+		err = sendAndVerifyMsgSend(app, mapper, msg, ctx, []crypto.PrivKey{fromAcc.PrivKey}, handler)
+		if err != nil {
+			return "", nil, err
```

Comment:
Should the action here be something like `bank/sendAndVerifyTxSend/false` @cwgoes?

Reply:
if we always construct a tx that will succeed, this is OK, but I think it is better to construct txs which will sometimes fail